Packs/JARM/Integrations/JARM/JARM.py

```diff
@@ -0,0 +1,159 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from urllib.parse import urlparse
+from ipaddress import ip_address
+from typing import Dict, Tuple, Any
+from jarm.scanner.scanner import Scanner
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+DEFAULT_PORT = 443
+
+""" CLIENT CLASS """
+
+
+class Client:
+    def jarm_fingerprint(self, host: str, port: int) -> Tuple[str, str, int]:
+        return Scanner.scan(host, port)
```

Comment:
Do you think we should add any exception handling to catch things like unresolvable domains?

Reply:
That should be handled by the pyJARM library that will throw an exception. The integration should be minimal.